web3/contract.py

```diff
@@ -766,6 +766,30 @@ def check_forbidden_keys_in_transaction(transaction, forbidden_keys=None):
             raise ValueError("Cannot set {} in transaction".format(', '.join(keys_found)))
 
 
+class ConciseMethod:
+    ALLOWED_MODIFIERS = set(['call', 'estimateGas', 'transact', 'buildTransaction'])
```

Comment:
Since we're touching this code anyway, how about:

```py
ALLOWED_MODIFIERS = {'call', 'estimateGas', 'transact', 'buildTransaction'}
```

Reply:
:+1: did not know about this syntax